leshan-server-demo/src/main/java/org/eclipse/leshan/server/demo/LeshanServerDemo.java

```diff
@@ -158,7 +160,11 @@ public static LeshanServer createLeshanServer(LeshanServerDemoCLI cli) throws Ex
         } else {
             // use Redis Store
             securityStore = new RedisSecurityStore(cli.main.redis);
-            builder.setRegistrationStore(new RedisRegistrationStore(cli.main.redis));
+            RedisRegistrationStore.Builder redisregistrationstorebuilder = new RedisRegistrationStore.Builder(
+                    cli.main.redis);
+            redisregistrationstorebuilder.setIdentitySerDes(new LwM2mIdentitySerDes());
+            redisregistrationstorebuilder.setPeerSerDes(new LwM2mPeerSerDes());
+            builder.setRegistrationStore(redisregistrationstorebuilder.build());
```

Comment:
This seems not really needed right ?

Reply:
You're right - it's part of my PoC - already fixed in the latest commit